src/main/scala/com/gd/twitteranalytics/reports/ReportInputDataParser.scala

```diff
@@ -4,16 +4,15 @@ import org.apache.spark.sql.{DataFrame, SparkSession}
 
 object  ReportInputDataParser{
 
-  def getPayloadStatusAsDataset(spark:SparkSession,path:String):DataFrame ={
+  def getPayloadStatusAsDataFrame(spark:SparkSession,path:String):DataFrame ={
```

Comment:
As you can see, subject of the commit is too long, so it doesn't get displayed properly.
please follow the guide for proper commits. (the guide from the reading list)

Reply:
This particular commit message is inserted by GitHub